lib/broccoli/app-config.js

```diff
@@ -1,7 +1,3 @@
-/* jshint ignore:start */
```

Comment:
Some files are ignored because they are not valid JS and they trip up the parser